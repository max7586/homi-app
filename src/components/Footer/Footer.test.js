import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Footer from './Footer';
import { BrowserRouter } from 'react-router-dom'

describe('Footer component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
        <BrowserRouter>
        <Footer />
      </BrowserRouter>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();  
  });


});