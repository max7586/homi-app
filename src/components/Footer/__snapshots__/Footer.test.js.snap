// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders the UI as expected 1`] = `
<div
  className="footer"
>
  <p>
    homi real estate | 2019 Ahmed @ am.
  </p>
  <ul>
    <li>
      <a
        href="http://www.facebook.com"
        rel="noopener noreferrer"
        target="_blank"
      >
        <i
          className="fab fa-facebook"
        />
      </a>
    </li>
    <li>
      <a
        href="http://www.twitter.com"
        rel="noopener noreferrer"
        target="_blank"
      >
        <i
          className="fab fa-twitter"
        />
      </a>
    </li>
    <li>
      <a
        href="http://www.instagram.com"
        rel="noopener noreferrer"
        target="_blank"
      >
        <i
          className="fab fa-instagram"
        />
      </a>
    </li>
  </ul>
</div>
`;
